setup.cfg

```diff
@@ -1,4 +1,7 @@
 [flake8]
-# 201 and 202 are spaces after ( and before ), 501 is line length. These
-# are exceptions allowed (encouraged?) by Galaxy style guidelines.
-ignore = E201,E202,E501
+# These are exceptions allowed (encouraged?) by Galaxy style guidelines.
+# 128 continuation line under-indented for visual indent
+# 201 and 202 are spaces after ( and before )
+# 203 whitespace before ':'
```

Comment:
We've agreed that we break with E201, E202, and E501.  E128 and E203 are a compromise to relax enforcement across the board, but there hasn't been an agreement to encourage anything other than 'make it readable'.